tools/debug.sh

```diff
@@ -42,10 +43,10 @@ case $1 in
         SYSTEM=$2
 
         declare -a SWITCHES=("switchtec0" "switchtec1")
-        for SWITCH in ${SWITCHES[@]};
+        for SWITCH in "${SWITCHES[@]}";
         do
             echo "$SWITCH Capturing Logs..."
-            ssh root@$SYSTEM <<-EOF
+            ssh root@"$SYSTEM" <<-EOF
```

Comment:
Why are quotes need?

Reply:
Shellcheck wants them quoted to avoid work splitting.

Reply:
Is that a vscode plugin?